composer.json

```diff
@@ -9,24 +9,22 @@
     "license": "MIT",
     "require": {
         "php": "^8.0.2",
-        "opcodesio/log-viewer": "^1.2",
-        "fruitcake/laravel-cors": "^2.0",
+        "opcodesio/log-viewer": "^1.9",
         "guzzlehttp/guzzle": "^6.0|^7.0",
-        "laravel/framework": "^9.0",
-        "laravel/sanctum": "^2.9",
+        "laravel/framework": "^10.0",
+        "laravel/sanctum": "^3.2",
         "laravel/telescope": "4.x-dev",
         "laravel/tinker": "^2.5",
-        "laravel/ui": "^3.0",
         "spatie/laravel-littlegatekeeper": "1.1",
         "spatie/laravel-query-builder": "^5.0"
```

Comment:
Let's upgrade to 5.2 
https://github.com/spatie/laravel-query-builder/tree/main